cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -59,7 +61,7 @@ func (r *Reporter) EmitZipkinBatch(zSpans []*zipkincore.Span) error {
 	if err != nil {
 		return err
 	}
-	return r.send(trace.Spans, nil)
+	return r.send(addAgentTags(trace.Spans, r.agentTags), nil)
```

Comment:
seems not DRY, can do this in one place, in send()

Reply:
Will do.